internal/repository/root.go

```diff
@@ -60,37 +73,47 @@ func (r *Repository) AddTopLevelTargetsKey(ctx context.Context, rootKeyBytes, ta
 	if err != nil {
 		return err
 	}
+	slog.Debug("Loaded new signer verifier in custom ssl format to generate key id of root of trust")
 	rootKeyID, err := sv.KeyID()
 	if err != nil {
 		return err
 	}
+	slog.Debug("Created root key id with signer verifier")
 
 	state, err := policy.LoadCurrentState(ctx, r.r)
 	if err != nil {
 		return err
 	}
+	slog.Debug("Loaded current state of repository's active policy")
 
 	rootMetadata, err := state.GetRootMetadata()
 	if err != nil {
 		return err
 	}
+	slog.Debug("Constructed root meta data with deserialized paylod of state's root envelope")
 
 	if !isKeyAuthorized(rootMetadata.Roles[policy.RootRoleName].KeyIDs, rootKeyID) {
 		return ErrUnauthorizedKey
 	}
+	slog.Debug("Successfully authorized root of trust with root key id")
```

Comment:
We are not authorizing the root key here. Just checking if it is already authorized or not.

Reply:
```suggestion
```